arch/arm/boot/dts/xilinx/ni-zynq.dtsi

```diff
@@ -235,38 +114,41 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 		};
+	};
 
-		spi0: spi@e0006000 {
-			compatible = "cdns,spi-r1p6";
-			status = "disabled";
-			reg = <0xe0006000 0x1000>;
-			interrupts = <0 26 4>;
-			clock-names = "ref_clk", "pclk";
-			clocks = <&clkc 25>, <&clkc 34>;
-			#address-cells = <1>;
-			#size-cells = <0>;
-		};
+};
 
-		spi1: spi@e0007000 {
-			compatible = "cdns,spi-r1p6";
-			status = "disabled";
-			reg = <0xe0007000 0x1000>;
-			interrupts = <0 49 4>;
-			clock-names = "ref_clk", "pclk";
-			clocks = <&clkc 26>, <&clkc 35>;
-			#address-cells = <1>;
-			#size-cells = <0>;
-		};
+&smcc {
+	status = "okay";
+};
 
-		usb0: usb@e0002000 {
-			usb-phy = <&usb_phy0>;
-		};
+&nfc0 {
+	status = "okay";
 
-		usb1: usb@e0003000 {
-			usb-phy = <&usb_phy1>;
```

Comment:
Same comments as above. In general, make sure all the stuff you are removing actually has an equivalent in the common zynq-7000.dtsi we are including and is not just being extended here.